code/dataaccess/getdata.q

```diff
@@ -1,10 +1,16 @@
 // high level api functions for data retrieval
-
-
-getdata:{[inputparams]
-  if[.proc.proctype in key inputparams;inputparams:inputparams .proc.proctype];
+\d .dataaccess
+getprocdata:{[inputparams]
+  // if process is striped procname will be in the key
+  if[a:.proc.procname in key inputparams;inputparams:inputparams .proc.procname];
+  if[not[a]&.proc.proctype in key inputparams;inputparams:inputparams .proc.proctype];
+  // optimize hdb query from the gateway
+  if[`optimhdb in key inputparams;
```

Comment:
is `optimhdb` generated elsewhere in code or are users exepcted to input this somewhere when using data access API? if the latter, it should have a more user-friendly name IMO

Reply:
It's a user input for the Data Access API - I believe its used for optimising hdb queries in which the data to query is across an entire day or more as the code will covert the starttime/endtime arguments to query into dates. We will look at this and add it into the documentation (and also rename to something like `optimisehdb`)

Reply:
if it's existing in data access API prior to TorQ cloud, don't worry about changing it, but yeah if it's added, please give it a nicer name

Reply:
Found it's actually generated in the code, not an input as previously thought.  Renaming it to make the purpose of it more clear regardless.